### PR TITLE
Modify schema-loader-command-name

### PR DESCRIPTION
**Pull request by brfrn169**
This PR modifies the schema loader command name. Please take a look!

### CODE REVIEW

File: schema-loader/src/main/java/com/scalar/db/schemaloader/command/SchemaLoaderCommand.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import picocli.CommandLine.Option;
 
 @Command(
-    name = "java -jar scalardb-schema-loader-<version>.jar",
+    name = "scalardb-schema-loader",
```
Comment (brfrn169) [inline schema-loader/src/main/java/com/scalar/db/schemaloader/command/SchemaLoaderCommand.java:20]: We don't necessarily use it with `java -jar` actually.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Pull request merged**